packages/neutrino/bin/neutrino

```diff
@@ -88,17 +88,9 @@ function run(command, presets) {
       `Ensure this module can be found relative to the root of the project.`);
   });
 
-  // Also grab any Neutrino config from package.json and merge it into the config at a higher precedence
-  if (Object.keys(config).length) {
-    api.config.merge(config);
-  }
-
-  if (args.inspect) {
-    return inspect(api.getWebpackOptions());
-  }
-
-  api[command](args)
-    .then(() => process.exit(0))
+  api.load()
+    .then(() => api.config.merge(config || {}))
+    .then(() => args.inspect ?  inspect(api.getWebpackOptions()) : api[command](args))
```

Comment:
Hmmmm... I'm not 100% in love with breaking them up. For someone else consuming the code, I don't think it is immediately apparent that inspect results in an exit.

Keeping them together in some sort of if/else or ternary shows that execution is one or the other. Ultimately your call. Just my 2 cents.